ts/utils/login.ts

```diff
@@ -23,7 +24,7 @@ type LoginFailure = {
 
 type LoginResult = LoginSuccess | LoginFailure;
 
-export type IdpTypes = "SPID" | "CIE_ID" | "CIE";
+type IdpTypes = keyof IdpData | "cie" | "cieid";
```

Comment:
"cie" is already present on type IdpData
```suggestion
type IdpTypes = keyof IdpData | "cieid";
```

Reply:
Addressed [here](https://github.com/pagopa/io-app/pull/6428/commits/b5296ea5aef56df65a6d3cd29dab15f81242319f). The `cieid` idp type has been added too in [this PR](https://github.com/pagopa/io-services-metadata/pull/864)